### PR TITLE
Solve tests timeout

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

File: .papr.sh [hunk 1]
```diff
@@ -52,6 +52,8 @@ if test -z "${INSIDE_CONTAINER:-}"; then
     exit 0
 fi
 
+echo "fastestmirror=true" >> /etc/dnf/dnf.conf
```
Comment (yuqi-zhang) [inline .papr.sh]: Is this the relevant line?

**Comment by yuqi-zhang** [comment]
cc @jlebon

**Comment by giuseppe** [comment]
@jlebon ok thanks for the feedback, I am going to close this PR.  The tests in the other PRs seem to pass now

**Pull request closed without merging**